Dockerfile

```
@@ -9,12 +9,26 @@ RUN curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel
     && echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list \
     && apt update && apt install -y bazel
 
-WORKDIR /app
+RUN groupadd -g 1000 myuser && \
+    useradd -m -u 1000 -g 1000 -s /bin/bash myuser
 
-ENV BAZEL_OPTS="--enable_workspace=true"
+USER myuser
 
-COPY . .
+RUN mkdir -p /home/myuser/squares
+
```

Encapsulate and export paths and reuse it during image build, and those environment variables can be re-used during upcoming development
```suggestion
USER_DIR=/home/user
REPOSITORY_DIR=/workspaces/SquareTutorial
```

---

fixed